gemrb/GUIScripts/iwd2/MessageWindow.py

```diff
@@ -38,8 +39,6 @@ def OnLoad():
 	OptionsWindow = MessageWindow = GemRB.LoadWindow(0)
 	ActionsWindow = PortraitWindow = GUICommonWindows.OpenPortraitWindow()
 
-	GemRB.SetVar ("MessageWindow", MessageWindow.ID)
-	GemRB.SetVar ("PortraitWindow", PortraitWindow.ID)
 	GemRB.SetVar ("ActionsWindow", 12345) # dummy so the EF_ACTION callback works
 	GemRB.SetVar ("TopWindow", -1)
 	GemRB.SetVar ("OtherWindow", -1)
```

Comment:
None of these `SetVar` statements are needed anymore. What is needed instead is to assign the appropriate aliases to the windows that `Interface::HandleEvents` looks for. See the BG2 MessageWindow.py.